lang/nl/lang.php

```diff
@@ -43,7 +43,7 @@
         'scan_messages_success' => 'De thema bestanden zijn succesvol gescand!',
         'scan_messages_hint' => 'Klikken op <strong>Zoeken naar nieuwe berichten</strong> controleert het huidige thema bestanden voor nieuwe berichten om te vertalen. ',
```

Comment:
This is not a good sentence.

Reply:
Yes sounds better!

Reply:
I'll produce another PR for it :) @LukeTowers is to fast :)